config/database.json

```diff
@@ -8,14 +8,14 @@
   },
   "test": {
     "username": "root",
-    "password": null,
+    "password": "password",
     "database": "gyro_test",
     "host": "127.0.0.1",
     "dialect": "mysql"
   },
   "production": {
     "username": "root",
-    "password": null,
+    "password": "password",
```

Comment:
I guess we don't have to do one for the production environment. I really just want to do it for the test environment. I will change this back to null.